test/Generic_JDBC_Tests/package.yaml

```diff
@@ -5,3 +5,4 @@ license: MIT
 author: enso-dev@enso.org
 maintainer: enso-dev@enso.org
 prefer-local-libraries: true
+jvm: true
```

Comment:
> cli now also respects package.yaml configuration. `Generic_JDBC_Tests` now serves as a test - it would fail in NI but now it passes.

Should you modify [context.rs](https://github.com/enso-org/enso/blob/94d48dac7dcb4fdc1ca2d7c0b91b7941ad4e9231/build_tools/build/src/engine/context.rs#L718) (where all test projects executed by NI are listed) to also include `Generic_JDBC_Tests`?

Reply:
Contrary to what one might expect, those aren't run anyway. See my [other PR](https://github.com/enso-org/enso/pull/12531) that fixes it.